esp32c6/svd/patches/esp32c6.yaml

```diff
@@ -76,10 +96,81 @@ I2C0:
       FIFO_RDATA:
         access: read-write
 
+_modify:
+  I2S0:
+    name: I2S
```

Comment:
If it's going to make implementing the driver more difficult please feel free to disregard this and we can address it at a later date.

I think I'd prefer to keep this as `I2S0`. I checked the other PACs, in `esp32` and `esp32s3` they are called `I2S0` but in `esp32c3` and `esp32s2` they are called `I2S`; I think `I2S0` is more in line with our naming style, and also in the case there are multiple `I2S` peripherals (such as on the ESP32-S3) we need to use this naming convention anyway.

Reply:
Thanks! We can easily go with I2S0